bin/faucet/src/server/get_tokens.rs

```diff
@@ -52,6 +61,12 @@ pub enum InvalidRequest {
     AccountId(#[source] AccountIdError),
     #[error("asset amount {0} is not one of the provided options")]
     AssetAmount(u64),
+    #[error("invalid POW solution")]
+    InvalidPoW,
+    #[error("invalid server signature")]
+    InvalidServerSignature,
+    #[error("server timestamp expired")]
+    ExpiredServerTimestamp,
```

Comment:
Maybe this could include the received timestamp and the current timestamp? Should make it easier to troubleshoot/debug when reading messages.